classes/label.rb

```diff
@@ -1,16 +1,16 @@
 class Label
-  attr_accessor :title
-  attr_reader :items
+  attr_accessor :title, :color
+  attr_reader :items, :id
 
-  def initialize(_id, title, color, _items)
+  def initialize(title, color, items)
     @id = Random.rand(1..100)
     @title = title
     @color = color
-    @items = []
+    @items = items
   end
```

Comment:
Here is the best way to set up the association between Label and Item classes.

`def initialize(title, color)
    @id = Random.rand(1..100)
    @title = title
    @color = color
    @items = []
  end
`

Reply:
Thank you @kensteph for your review!